File: Python/1065.py
pares = []
lines = []
for i in range(5):
    a = int(input())
    lines.append(a)
    
for j in range(len(lines)):
    if (lines[j] % 2 == 0):
        pares.append(j) 
print(str(len(pares)) + " valores pares")

File: beecrowd/javascript/1065.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n').filter(Boolean).map(l => parseInt(l));

pares = []
for (let i = 0; i<lines.length; i++){
    if (lines[i] % 2 === 0) {
        pares.push(lines[i])
    }
}
 console.log(`${pares.length} valores pares`)